Examples/Swift/FillPatternExample.swift

```diff
@@ -0,0 +1,59 @@
+import Mapbox
+
+@objc(FillPatternExample_Swift)
+
+class FillPattern_Swift: UIViewController, MGLMapViewDelegate {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Set the map’s size, style, center coordinate, and zoom level.
+        let mapView = MGLMapView(frame: view.bounds)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.styleURL = MGLStyle.darkStyleURL(withVersion: 9)
+        mapView.setCenter(CLLocationCoordinate2D(latitude: 38.849534447, longitude: -77.039222717), zoomLevel: 8.5, animated: false)
+        
+        view.addSubview(mapView)
+        
+        mapView.delegate = self
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Set the UIImage to be used for the fill pattern.
+        let fillPatternImage = UIImage(named: "stripe-pattern")!
+        
+        // Add the fill pattern image to used by the style layer.
+        style.setImage(fillPatternImage, forName: "stripe-pattern")
+        
+        // "mapbox://examples.0cd7imtl" is a map ID referencing a tileset containing vector data.
+        // For more information, see mapbox.com/help/define-map-id/
+        let source = MGLVectorSource(identifier: "drone-restrictions", configurationURL: URL(string: "mapbox://examples.0cd7imtl")!)
+        style.addSource(source)
+        
+        // Create a style layer using the vector source.
+        let layer = MGLFillStyleLayer(identifier: "drone-restrictions-style", source: source)
+        
+        // Set the source's identifier using the source name retrieved from its
+        // TileJSON metadata: mapbox.com/api-documentation/#retrieve-tilejson-metadata
```

Comment:
Mapbox Studio provides a user-friendly way to see these layers without having to construct an API request by hand.